src/main/resources/application.yml

```diff
@@ -16,6 +16,24 @@ spring:
           maxConnectionLifeTimeMS: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTION_LIFE_MS:0}
           maxConnectionIdleTimeMS: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTION_IDLE_MS:120000}
           maxConnecting: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTING:2}
+  cloud:
+    stream:
```

Comment:
in order to pre-load the producer, we should define the function as done here:
https://github.com/pagopa/idpay-payment/blob/main/src/main/resources/application.yml#L28